x/auth/keeper/grpc_query.go

```diff
@@ -224,7 +228,16 @@ func (s queryServer) AccountInfo(ctx context.Context, req *types.QueryAccountInf
 
 	account := s.k.GetAccount(ctx, addr)
 	if account == nil {
-		return nil, status.Errorf(codes.NotFound, "account %s not found", req.Address)
+		xAccount, err := s.getFromXAccounts(ctx, addr)
+		if err != nil {
```

Comment:
nit: if we don't return the error here, we can collapse to `if err != nil || xAccount.Info == nil`

Reply:
done!